src/utils/PreparePeopleHelper.ts

```diff
@@ -0,0 +1,63 @@
+import { Person } from '../types';
+import { SortParam } from '../types/SortParam';
+
+type FilterPeopleType = (
+  people: Person[],
+  gender: string | null,
+  query: string | null,
+  century: string[],
+) => Person[];
+
+export const FilterPeople: FilterPeopleType = (
+  people,
+  gender,
+  query,
+  century,
+) => {
```

Comment:
The function that takes 3 or more arguments is hard to maintain. It's really easy to mess up the order of params, so you should pass an object.